luma/core/device.py

```diff
@@ -90,6 +91,61 @@ def cleanup(self):
         self._serial_interface.cleanup()
 
 
+class parallel_device(device):
+    """
+    Wrapper class to manage communications with devices that can operate in
+    four or eight bit modes.
+
+    .. versionadded:: 1.15.1
+    """
+
+    def __init__(self, const=None, serial_interface=None, exec_time=1e-6 * 50, **kwargs):
```

Comment:
can you document the constructor vars, e.g why that value for `exec_time`?

Reply:
Do you want this to show up in the documentation?  The class is only intended to be used by the drivers that require 4 bit handling which right now is the hd44780 and the ws0010/winstar_weh drivers.  I would not consider it a developer facing class.  Happy to add the documentation but wanted to see how you preferred handling.

Reply:
Well, i understand what you mean with it showing up in the docs but I don't like these magic nrs without documentation.

Reply:
maybe a documented module-level constant?

Reply:
it's also used here: https://github.com/rm-hull/luma.core/blob/master/luma/core/interface/serial.py#L520

Reply:
I eliminated the default value for exec_time here.  It really isn't parallel_device's responsibility.  It's just pushing it along from the device which is where the default should really be established.  I will document the reason for the value on those classes (within luma.lcd.device.hd44780 and luma.oled.device.ws0010).  I also moved the default for pulse_time in luma.core.interface.serial.parallel (and pcf8574) to a module level variable and placed some documentation above it.  Let me know if you'd like it handled differently.

Reply:
looks great!